cypress/integration/Buttons.spec.js

```diff
@@ -0,0 +1,29 @@
+/// <reference types="cypress" />
+
+describe('Buttons Screen', () => {
+  beforeEach('visits site', () => {
+    cy.visit('http://localhost:6006')
+    cy.contains('Buttons', { timeout: 20000 })
+  })
+  it('Displays the learn button', () => {
+    cy.contains('Buttons')
+    cy.visit(
+      'http://localhost:6006/iframe.html?id=buttons--learnabled&args=trust:false&viewMode=story'
+    )
+    cy.contains('Learn more')
+  })
+  it('Displays the learn button', () => {
```

Comment:
This testcase description already exists above. Can you be much more detailed here